nap/docker/ckan/ckan-plugins/ckanext-napote_theme/ckanext/napote_theme/templates/header.html

```diff
@@ -145,6 +146,11 @@
                                 </ul>
                             {% else %}
                                 <ul class="nav nav-pills">
+                                    <li>
+                                        <a href="https://s3.eu-central-1.amazonaws.com/ote-assets/nap-ohje.pdf" target="_blank" title="NAP-palvelun käyttöohje">
```

Comment:
here as well

Reply:
I Copy-pasted the old implementation. I'll add the missing translation key.